UnitTests/UnitTests.UWP/Extensions/Test_LogicalTreeExtensions.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Toolkit.Uwp.Extensions;
+using Microsoft.Toolkit.Uwp.UI.Extensions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Markup;
+
+namespace UnitTests.Extensions
+{
+    [TestClass]
+    public class Test_LogicalTreeExtensions : VisualUITestBase
+    {
+        [TestCategory("LogicalTree")]
+        [TestMethod]
+        public async Task Test_LogicalTree_FindParent_Exists()
+        {
+            await App.DispatcherQueue.EnqueueAsync(async () =>
+            {
+                var treeRoot = XamlReader.Load(@"<Page
+    xmlns=""http://schemas.microsoft.com/winfx/2006/xaml/presentation""
+    xmlns:x=""http://schemas.microsoft.com/winfx/2006/xaml"">
+    <Grid>
+        <Grid> <!-- Target -->
+            <Border/>
+            <Border>
+                <TextBlock/> <!-- Starting Point -->
+            </Border>
+        </Grid>
+    </Grid>
+</Page>") as Page;
```

Comment:
If this is something you might use more often, consider creating a dedicated UserControl/Page in the test project instead of using XAML reader often.

Reply:
Yeah, that's how we have things setup in the Integration tests. Here these are more isolated scenarios specific to the test, so I think it makes sense to keep them inline for reference.

Reply:
Just wanted to mention that given in a future PR you might want to test all functions on the same visual tree and it might be easier to have one "sample tree" instead of freshly loading it from XAML all the time.